core/variant/variant_parser.cpp

```diff
@@ -1423,15 +1423,15 @@ Error VariantWriter::write(const Variant &p_variant, StoreStringFunc p_store_str
 		} break;
 		case Variant::VECTOR2: {
 			Vector2 v = p_variant;
-			p_store_string_func(p_store_string_ud, "Vector2( " + rtosfix(v.x) + ", " + rtosfix(v.y) + " )");
+			p_store_string_func(p_store_string_ud, "Vector2(" + rtosfix(v.x) + ", " + rtosfix(v.y) + ")");
 		} break;
 		case Variant::VECTOR2I: {
 			Vector2i v = p_variant;
 			p_store_string_func(p_store_string_ud, "Vector2i( " + itos(v.x) + ", " + itos(v.y) + " )");
```

Comment:
Should do the same for `Vector2i` (and review other new types).

The docs will also need to be synced.

Reply:
I know, just a little busy right now. Wish that `doctool` could update those automatically...

Reply:
> Wish that `doctool` could update those automatically...

`doctool` would upload the docs automatically to change those default values normally (at least it works in the CI which then treats it as an error).

Reply:
> `doctool` would upload the docs automatically to change those default values normally

I don't understand, you mean it would do it when I `push`?

Reply:
You should call doctool manually with a compiled binary corresponding to this PR (`bin/godot.blabla --doctool`). Then you should get exactly what is in https://github.com/godotengine/godot/pull/32313/checks?check_run_id=2853705737

If you don't, you're probably running the wrong Godot build.

Reply:
I know how to call it, and there's only one build of Godot in the folder, so there's that.

Well, nothing that a little regex can't solve.

Reply:
I'm still confused that `doctool` wouldn't work for you. Did you build your PR and use the Godot build from this PR? If it only changed stuff in `CanvasItem`, it means you have a slightly older build of `master` without this PR and with some `CanvasItem` stuff not included yet (or changed since).

As a reminder, `doctool` does not parse the current codebase, it extracts the information which has been compiled into the binary.